styles/modal.css

```diff
@@ -0,0 +1,23 @@
+.modal {
+  border: 2px solid var(--brand-secondary);
+  max-width: 22rem;
+  border-radius: 5px;
+  padding: 10px;
+  z-index: 1;
```

Comment:
Z index won't be effective without position. Either define position or else you can remove this property.

Reply:
ok